config_scenario1.py

```diff
@@ -0,0 +1,64 @@
+"""Example configuration file
+
+This file aims to set parameters to realistic values
+"""
+
+import logging
+import itertools
+import numpy as np
+from DAS.shape import Shape
+
+dumpXML = 1
+visualization = 1
+logLevel = logging.INFO
+
+# number of parallel workers. -1: all cores; 1: sequential
+# for more details, see joblib.Parallel
+numJobs = -1
+
+# distribute rows/columns evenly between validators (True)
+# or generate it using local randomness (False)
+evenLineDistribution = False
+
+# Number of simulation runs with the same parameters for statistical relevance
+runs = range(1)
+
+# Number of validators
+numberNodes = [8000]
+
+# Percentage of block not released by producer
+failureRates = [0]
+
+# Block size in one dimension in segments. Block is blockSizes * blockSizes segments.
+blockSizes = [512]
+
+# Per-topic mesh neighborhood size
+netDegrees = [6]
+
+# number of rows and columns a validator is interested in
+chis = [2]
+
+# ratio of class1 nodes (see below for parameters per class)
+class1ratios = [0.8]
+
+# number of validators per beacon node
+validatorsPerNode1 = [1]
+validatorsPerNode2 = [100]
```

Comment:
I think 500 is more realistic, based on Lido's report. Also, is the value recommended by Dankrad.